src/pages/components/avatar.mdx

```diff
@@ -63,20 +62,23 @@ import AvatarStatusCode from '!!raw-loader!../../examples/avatar/AvatarStatus.ts
   <AvatarStatus />
 </CodeExample>
 
-## Best practices
+## How to use it well
 
-### Selecting the right Avatar shape
+### Select the right shape for the job
 
-Circular Avatars are used to represent a person because the shape is most often associated with a
-profile picture. Rounded squares are used for system applications like objects, brands, or products.
-This helps the user quickly distinguish between a person or a system when scanning.
+Use circles to represent people and rounded squares for objects, brands, or products. This helps a
+user distinguish between the two when scanning a page.
 
 <BestPractice>
   <Do imageSource={props.data.avatarShapeCircle.children[0].publicURL}>
-    <p>Use a circular shape to represent a person.</p>
+    <ul>
+      <li>Use a circular Avatar for people.</li>
+    </ul>
   </Do>
   <Do imageSource={props.data.avatarShapeSquare.children[0].publicURL}>
-    <p>Use the rounded square shape to represent objects, brands, or products.</p>
+    <ul>
+      <li>Use a rounded square Avatar for objects, brands, or Zendesk products.</li>
+    </ul>
```

Comment:
Why were these changes made? When the information is not a list, a `<p>` is the more semantic HTML.